docs/02-guides/analytics/custom-plugins.mdx

```diff
@@ -0,0 +1,224 @@
+---
+sidebar_position: 3
+title: Custom Plugins
+description:
+  The analytics library is designed to work with any third party analytics tools
+  via plugins, in this guide we will explore how to write and use custom
+  plugins.
+---
+
+import AnalyticsFigure from "./assets/analytics.svg";
+import Figure from "@site/src/components/Figure";
+import ContactLink from "@site/src/components/ContactLink";
+import { useState } from "react";
+import SinceVersion from "@site/src/components/SinceVersion";
+
+<SinceVersion tag="2.19" />
```

Comment:
**issue:** it's not part of the 3.x branch.

**suggestion:** The `<SinceVersion>` from 2.x should be removed